src/SpinWaveTheory/HamiltonianDipole.jl

```diff
@@ -20,7 +20,7 @@ function swt_hamiltonian_dipole!(H::Matrix{ComplexF64}, swt::SpinWaveTheory, q_r
     (; extfield, gs, units) = sys
     for i in 1:L
         B = units.μB * (gs[1, 1, 1, i]' * extfield[1, 1, 1, i]) 
-        B′ = dot(B, local_rotations[i][:, 3]) / 2 
+        B′ = - dot(B, local_rotations[i][:, 3]) / 2
```

Comment:
As we discussed, it's a bit strange to me too that this is necessary (as far as consistency with SU(N) Hamiltonian construction), but it seems to result in consistent results when tested on a chiral model.

Reply:
The lack of a minus sign in dipole-LSWT code (prior to this PR) also seemed strange to me. I had a phone call with Hao, and he explained that it's correct, and has to do with the way that the spin operator is encoded in bosons. Hao said the formula is something like: $\hat{S} \sim - b^\dagger b$, which is the origin of the minus sign appearing in the Hamiltonian construction.